integration/log.go

```diff
@@ -24,6 +24,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/cenkalti/backoff"
```

Comment:
I noticed that package, but this one seems significantly better. Is there a particularly good reason that we implemented backoff ourselves?